src/js/components/Menu/stories/CustomThemed/Themed.tsx

```diff
@@ -60,10 +60,16 @@ const customBreakpoints: ThemeType = {
     },
   },
   menu: {
+    background: 'light-3',
     icons: {
       down: FormDown,
       up: FormUp,
     },
+    button: {
+      color: 'brand',
+      style: { margin: '6px' },
+    },
+    extend: `border: 3px solid gray`,
```

Comment:
Why `extend` here instead of just `theme.menu.border`?

Reply:
Right! That would be nice if we had `theme.menu.border` :) It could be a nice enhancement request but not pressing on my end.